suite-native/module-device-settings/src/screens/DeviceSettingsModalScreen.tsx

```diff
@@ -27,6 +28,7 @@ export const DeviceSettingsModalScreen = () => {
     const [isUpdateSheetOpen, setIsUpdateSheetOpen] = useState<boolean>(false);
 
     const isUpgradable = deviceReleaseInfo?.isNewer ?? false;
+    const [isFirmwareUpdateEnabled] = useFeatureFlag(FeatureFlag.IsFirmwareUpdateEnabled);
```

Comment:
I think it is not necessary to have here, this should be accessible if FF is off anyway

Reply:
true, fixed